### PR TITLE
examples and further notes about duplicate policy #49

### PR DESCRIPTION
**Pull request by AvitalFineRedis**

### CODE REVIEW

File: NRedisTimeSeries/TimeSeriesClient.cs [hunk 1]
```diff
@@ -26,9 +26,9 @@ public static partial class TimeSeriesClient
         /// You can alter the default TSDB chunk size by passing the chunk_size argument (in Bytes)</param>
         /// <param name="policy">Optinal: Define handling of duplicate samples behavior (avalible for RedisTimeseries >= 1.4)</param>
         /// <returns>If the operation executed successfully</returns>
-        public static bool TimeSeriesCreate(this IDatabase db, string key, long? retentionTime = null, IReadOnlyCollection<TimeSeriesLabel> labels = null, bool? uncompressed = null, long? chunkSizeBytes = null, TsDuplicatePolicy? policy = null)
+        public static bool TimeSeriesCreate(this IDatabase db, string key, long? retentionTime = null, IReadOnlyCollection<TimeSeriesLabel> labels = null, bool? uncompressed = null, long? chunkSizeBytes = null, TsDuplicatePolicy? duplicatePolicy = null)
```
Comment (filipecosta90) [inline NRedisTimeSeries/TimeSeriesClient.cs:29]: Is this a backwards compatible change?

File: NRedisTimeSeries/TimeSeriesClient.cs [hunk 2]
```diff
@@ -64,9 +64,9 @@ public static bool TimeSeriesAlter(this IDatabase db, string key, long? retentio
         /// You can alter the default TSDB chunk size by passing the chunk_size argument (in Bytes)</param>
         /// <param name="policy">Optioal: overwrite key and database configuration for DUPLICATE_POLICY</param>
         /// <returns>The timestamp value of the new sample</returns>
-        public static TimeStamp TimeSeriesAdd(this IDatabase db, string key, TimeStamp timestamp, double value, long? retentionTime = null, IReadOnlyCollection<TimeSeriesLabel> labels = null, bool? uncompressed = null, long? chunkSizeBytes = null, TsDuplicatePolicy? policy = null)
+        public static TimeStamp TimeSeriesAdd(this IDatabase db, string key, TimeStamp timestamp, double value, long? retentionTime = null, IReadOnlyCollection<TimeSeriesLabel> labels = null, bool? uncompressed = null, long? chunkSizeBytes = null, TsDuplicatePolicy? duplicatePolicy = null)
```
Comment (filipecosta90) [inline NRedisTimeSeries/TimeSeriesClient.cs:67]: Is this a backwards compatible change?

File: README.md [hunk 3]
```diff
@@ -11,4 +11,57 @@
 
 .Net Client for RedisTimeSeries
 
+
+## API
+The complete documentation of RedisTimeSeries's commands can be found at [RedisTimeSeries's website](http://redistimeseries.io/).
+
+## Usage example
```
Comment (DvirDukhan) [inline README.md:18]: I think only the simple example should be in the readme. Have a link to the advanced API example files

File: README.md [hunk 4]
```diff
@@ -11,4 +11,57 @@
...
+db.TimeSeriesCreateRule("source", rule2);
+db.TimeSeriesAdd("source", "*", 1);
+db.TimeSeriesAdd("source", "*", 2);
+db.TimeSeriesAdd("source", "*", 3);
+db.TimeSeriesGet("source");
+db.TimeSeriesGet("sumRule");
+db.TimeSeriesGet("avgRule");
+```
+
+## Further notes on back-filling time series
```
Comment (DvirDukhan) [inline README.md:49]: Same comment as before
maybe have a dedicated `md` file on the examples?

File: NRedisTimeSeries/TimeSeriesClient.cs (hunk 1 above)
Comment (DvirDukhan) [inline NRedisTimeSeries/TimeSeriesClient.cs:29]: please change the documentation of the functions after this rename
here and on the async api

**Review by DvirDukhan — CHANGES_REQUESTED** [review]

**Review by DvirDukhan — APPROVED** [review]

**Pull request merged**